client-rest/examples/rest.rs

```diff
@@ -0,0 +1,110 @@
+//! Example communication with this service
+
+use chrono::NaiveDate;
+use hyper::StatusCode;
+use hyper::{Body, Client, Method, Request};
+use std::time::Duration;
+use svc_cargo_client_rest::types::*;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    println!("NOTE: Ensure the server is running, or this example will fail.");
+
+    let url = "http://0.0.0.0:8000";
```

Comment:
Is it possible to get the port from the HOST_PORT env var? This way we can tell the service which port to listen on so we don't have clashes with docker images trying to bind on the same port when running simultaneously.

Reply:
I would need to add a `-e HOST_PORT=$(HOST_PORT` line to the `docker_run` command in `base.mk` to expose the container to the Makefile variable. Is this acceptable? I can make the tf-github PR

And to double check, you want to expose the HOST_PORT, not the DOCKER_PORT?

Reply:
Updated: https://github.com/Arrow-air/svc-cargo/pull/1/commits/d72ac472cd65f1e7f2dfe9cbfab9d98e018017c1

Opened PR for exposing HOST_PORT to docker_run